packages/filesystem/src/node/chokidar-filesystem-watcher.ts

```diff
@@ -55,12 +55,16 @@ export class ChokidarFileSystemWatcherServer implements FileSystemWatcherServer
                 ignored: options.ignored
             });
             watcher.once('ready', () => {
-                this.logger.info(`Started watching:`, paths)
-                resolve(watcherId)
+                this.logger.info(`Started watching:`, paths);
+                resolve(watcherId);
+            });
+            watcher.on('error', error => {
+                if (this.isWatchingError(error) && error.code === 'EPERM') {
+                    this.logger.warn(`Cannot watch file changes due to insufficient permissions. Skipping: ${error.filename}.`);
```

Comment:
Maybe they should all be warnings with the error code ?
There's no reason why EPERM should be different IMO